src/ButtonComponent/Button.tsx

```diff
@@ -0,0 +1,14 @@
+import cn from 'classnames';
+import React from 'react';
+
+import { ButtonProps } from './types';
+
+const Button = ({ content }: ButtonProps): JSX.Element => {
+  return (
+    <div>
+      <button className={cn(`py-med px-lrg rounded-med`)}>{content}</button>
```

Comment:
`classnames` is used to conditionally join class names together, I don't see this feature used here, so why use it?

Reply:
I'm going to use it in the feature